cmd/run.go

```diff
@@ -124,6 +124,10 @@ func Run(isDebug *bool) *cli.Command {
 				Name:  "exp-use-winget-for-uv",
 				Usage: "use powershell to manage and install uv on windows, on non-windows systems this has no effect.",
 			},
+			&cli.StringFlag{
+				Name:  "exclude",
```

Comment:
I'd go with `--exclude-tags` and make it a string array, just like the `tags` one

Reply:
@karakanb As far as I see tag is not defined as a string array in the existing implementation --> `&cli.StringFlag{
				Name:    "tag",
				Aliases: []string{"t"},
				Usage:   "pick the assets with the given tag",
			},` would you like me to change it to string array ?